Multisig/Assets.xcassets/illustrations/ico-no-collectibles.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "ico-no-collectibles.png",
+      "filename" : "↳ Icon.pdf",
```

Comment:
such a strange filename 🤷

Reply:
Honestly I just extract from Figma and use it without notice the name, I will just rename to avoid any troubles